data/packages/expl3.json

```diff
@@ -6448,9 +6532,15 @@
     {
       "name": "muskip_set:Nn"
     },
+    {
+      "name": "muskip_set:NV"
+    },
     {
       "name": "muskip_set:cn"
     },
+    {
+      "name": "muskip_set:cV"
+    },
```

Comment:
Similarly, these two function variants were missing from latex3 dtx files due to typos. They are already fixed in latex3 dev (https://github.com/latex3/latex3/commit/20fa9ac27aa5439fa5f5f6cfc913ec67efcaeeba), thus I manually add them here.